scripts/portfolio.js

```diff
@@ -40,7 +48,7 @@
             Project.loadAll(data);
             localStorage.projects = JSON.stringify(data);
             localStorage.eTag = eTag;
-            callback();
+            initIndexPage();
```

Comment:
This is what you should also be doing on line 45.  You should think of a more generic name here though like next or callback.